### ISSUE TITLE
[Feat] Add National Land Cover Dataset (NLCD) color table

### ISSUE THREAD

**Issue by baharmon** [OSGeo/grass#508]
**Add NLCD color table to default GRASS color tables**
While there is a Corine land cover color table, there is not a NLCD color table in the default color tables for GRASS GIS. I propose that we this color table. I have also included a natural earth style color table for NLCD. In general I think more hypsometric color tables would also be a nice addition. 

**NLCD Color Table**
```
0 255 255 255
11 72:109:162
12 231:239:252
21 225:205:206
22 220:152:129
23 241:1:0
24 171:1:1
31 179:175:164
41 108:169:102
42 29:101:51
43 189:204:147
51 175:150:60
52 209:187:130
71 237:236:205
72 208:209:129
73 164:204:81
74 130:186:157
81 221:216:62
82 174:114:41
90 187:215:237
95 113:164:193
```

**[NLCD Natural Color Table](http://www.shadedrelief.com/shelton/c.html)**
```
0 255 255 255
11 188:221:255
12 255 255 255
21 175:157:179
22 175:157:179
23 102:85:127
24 99:82:123
31 243:236:210
41 159:188:157
42 29:101:51
43 189:204:147
51 176:151:61
52 200:221:156
71 252:232:171
72 243:245:161
73 164:204:81
74 227:228:170
81 227:228:170
82 227:228:170
90 127:180:152
95 164:204:196
```

**NLCD Categories**
```
11|Open Water
12|Perennial Ice/Snow
21|Developed, Open Space
22|Developed, Low Intensity
23|Developed, Medium Intensity
24|Developed, High Intensity
31|Barren Land
41|Deciduous Forest
42|Evergreen Forest
43|Mixed Forest
51|Dwarf Scrub
52|Shrub/Scrub
71|Grassland/Herbaceuous
72|Sedge/Herbaceous
73|Lichens
74|Moss
81|Pasture/Hay
82|Cultivated Crops
90|Woody Wetlands
95|Emergent Herbaceuous Wetlands
```

**Comment by neteler** [comment]
Would you be willing to prepare a pull request?

**Comment by baharmon** [comment]
Will do. Working on it.

**Comment by petrasovaa** [comment]
Fixed some time ago with PR #518.
